doc/news/_preparation_next_release.md

```diff
@@ -357,6 +357,11 @@ removed due to:
 - no time in future to maintain
   _(Kurt Micheli)_
 
+### RGBColor
+
+- New plugin to validate hex formatted colors (e.g. #fff or #abcd) and normalize them to rgba (#ffffffff and #aabbccdd respectively). It also has support for named colors according to the [extended color keywords](https://www.w3.org/TR/css-color-3/#svg-color) from CSS3.
```

Comment:
This is not up-to-date anymore.

Reply:
I updated the text to indicate that the plugin normalizes to decimal rrggbbaa instead of hex. Is that what you meant?